ssh/tailssh/incubator.go

```diff
@@ -126,22 +127,7 @@ func (ss *sshSession) newIncubatorCommand() (cmd *exec.Cmd) {
 		if isShell {
 			incubatorArgs = append(incubatorArgs, "--shell")
 		}
-		// Only the macOS version of the login command supports executing a
```

Comment:
There's no need to do this outside of the actual incubator process, so this has moved into `tryLoginCmd`.